pkg/machinery/constants/constants.go

```diff
@@ -269,6 +269,9 @@ const (
 	// EtcdImage is the reposistory for the etcd image.
 	EtcdImage = "gcr.io/etcd-development/etcd"
 
+	// EtcdOOMScoreAdj is an oom_score_adj for the etcd container.
+	EtcdOOMScoreAdj = -998
```

Comment:
I would like to see either all of the`oom_score_adj` be a const, or all _not_. In other words `etcd` has a `const` but the others don't. I am a stickler for consistency, so admittedly this is a nit. I am thinking that a `const` isn't required just _yet_ since really the service definitions are the only places these are useful. So I would lean towards just having _not_ be a `const`, but I could live with either.

Reply:
Etcd is not part of the system. I thought that this value can be changed through api in future. But in real world no one wants to change it. 
I've removed this one-time-use constant.
Thanks.

Reply:
Ah, I didn't consider that. Something we can do in the future if we decide.